runbot/runbot.py

```diff
@@ -1473,8 +1473,15 @@ def fast_launch(self, branch_name=False, repo=False, **post):
             if last_build.state != 'running':
                 url = "/runbot/build/%s?ask_rebuild=1" % last_build.id
             else:
-                url = ("http://%s/login?db=%s-all&login=admin&key=admin%s" %
-                       (last_build.domain, last_build.dest, "&redirect=/web?debug=1" if not build.branch_id.branch_name.startswith('7.0') else ''))
+                if build.branch_id.branch_name[0] not in ('7', '8'):
+                    base_url = "http://%s/web/login?db=%s-all&login=admin&key=admin%s"
```

Comment:
The `/web/login` route does not accept a `key` parameter, so it's a bit confusing to pass it.